spec/src/main/asciidoc/WebServicesMetadata.adoc

```diff
@@ -943,2259 +807,1399 @@ header, and the mode is OUT.
+      <input message="tns:notifyTransfer"/>
+    </operation>
+    
+    <operation name="login" parameterOrder="parameters">
+      <input message="tns:login"/>
+      <output message="tns:loginResponse"/>
+    </operation>
+  </portType>
+  
+  <binding name="ExampleWebServiceImplServiceSoapBinding" type="tns:ExampleWebService">
+    <soap:binding style="document" transport="http://schemas.xmlsoap.org/soap/http"/>
+    <operation name="createCustomer">
+      <soap:operation soapAction="urn:createCustomer" style="document"/>
+      <input>
+        <soap:body parts="parameters" use="literal"/>
+        <soap:header message="tns:createCustomer" part="token" use="literal"/>
+      </input>
+      <output>
+        <soap:body parts="parameters" use="literal"/>
+      </output>
+    </operation>
+    
+    <operation name="notifyTransfer">
+      <soap:operation soapAction="urn:notifyTransfer" style="document"/>
+      <input>
+        <soap:body parts="parameters" use="literal"/>
+        <soap:header message="tns:notifyTransfer" part="token" use="literal"/>
+      </input>
+    </operation>
+
+    <operation name="login">
+      <soap:operation soapAction="urn:login" style="document"/>
+      <input>
+        <soap:body parts="parameters" use="literal"/>
+      </input>
+      <output>
+        <soap:body parts="parameters" use="literal"/>
+      </output>
+    </operation>
+  </binding>
+
+  <service name="ExampleWebServiceImplService">
+    <port binding="tns:ExampleWebServiceImplServiceSoapBinding" name="ExampleWebServiceSoapPort">
+      <soap:address
+           location="http://localhost:7001/ExampleWebServiceImpl/ExampleWebServiceImpl"/>
+    </port>
+</service>
 </definitions>
+----
 
-*__4.6 Annotation: javax.jws.HandlerChain +
-__4.6.1 Description*
-
-The @HandlerChain annotation associates the Web Service with an
-externally defined handler chain (JAX-WS 2.0 [5], Section 9).
-
-It is an error to combine this annotation with the @SOAPMessageHandlers
-annotation.
-
-The @HandlerChain annotation MAY be present on the endpoint interface
-and service +
-implementation bean. The service implementation bean’s @HandlerChain is
-used if +
-@HandlerChain is present on both.
-
-The @HandlerChain annotation MAY be specified on the type only. The
-annotation target includes METHOD and FIELD for use by JAX-WS 2.0 [5]. A
-JSR-181 Processor is REQUIRED to report an error if the @HanderChain
-annotation is used on a method.
-
-The @HandlerChain annotation contains the following member-values:
-
-[cols=",,",]
-|===
-a|
-Member-Value
+== References
 
-a|
-Meaning
+. JSR-175 A Metadata Facility for the Java(TM) Programming Language
+https://jcp.org/en/jsr/detail?id=175
 
-a|
-Default
+. Jakarta Deployment 1.7 
+https://jakarta.ee/specifications/deployment/1.7/
 
-a|
-File
+. XML Schema 1.0
+https://www.w3.org/TR/xmlschema-1/
 
-a|
-Location of the handler chain file. The location supports 2 formats.
+. Jakarta EE
+https://jakarta.ee/specifications/
```

Comment:
Jakarta EE 9 should be referenced: https://jakarta.ee/specifications/platform/9/

Reply:
Ok. resolved to https://github.com/eclipse-ee4j/jws-api/pull/69/commits/0b6c04287b2400cd4ccaeb1cd387e9f5c3535f1c